benchmark/README.md

```diff
@@ -1,46 +1,15 @@
 # Benchmarks
 
-The following graphs show the results of benchmarking match attempts using different PSI algorithms on various Google Cloud VM's [general-purpose virtual machines (VMs)](https://cloud.google.com/compute/docs/general-purpose-machines#n2-standard) like n2-standard-64. In each benchmark, the sender and receiver use the same model VM. The receiver has 50m records while the sender has varying datasets of 50m, 100m, 150m, 200m and 250m records. The BPSI used for these experiments has a false positive rate fixed at 1e-6. 
+The following scatter plots show the results of benchmarking match attempts using different PSI algorithms on Google Cloud n2-standard-64 [general-purpose virtual machines (VMs)](https://cloud.google.com/compute/docs/general-purpose-machines#n2-standard). For each benchmark, the sender and receiver use the same model VM. In the first plot, the receiver has 100m records while the sender has varying datasets of 50m, 100m, 200m, 300m, 400m and 500m records where as in the second plot, the sender's dataset is of 100m records while the receiver has varying datasets of 50m, 100m, 200m, 300m, 400m and 500m records. The BPSI used for these experiments has a false positive rate fixed at 1e-6. 
```

Comment:
> For each benchmark, the sender and receiver use the same model VM.
This makes it sound like the sender and the receiver are run in the same VM, which is not the case.
Do the dataset or records that you call here have an intersection size?